rdfhash/utils/graph.py

```diff
@@ -287,6 +292,22 @@ class OxiGraph(__Graph__):
 
     supports_named_graphs = True
 
+    @staticmethod
+    def _get_rdf_format(format_str):
+        """Convert MIME type string to pyoxigraph.RdfFormat object"""
+        format_map = {
+            "application/trig": pyoxigraph.RdfFormat.TRIG,
+            "application/n-quads": pyoxigraph.RdfFormat.N_QUADS,
+            "application/n-triples": pyoxigraph.RdfFormat.N_TRIPLES,
+            "text/turtle": pyoxigraph.RdfFormat.TURTLE,
+            "text/n3": pyoxigraph.RdfFormat.N3,
+            "application/rdf+xml": pyoxigraph.RdfFormat.RDF_XML,
+            "application/ld+json": pyoxigraph.RdfFormat.JSON_LD,
+        }
+        return format_map.get(
+            format_str, pyoxigraph.RdfFormat.from_media_type(format_str)
```

Comment:
If `pyoxigraph` already provides the `from_media_type` method, are you overriding some of the choices?